src/App.js

```diff
@@ -1,25 +1,88 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from 'react'
+import './App.css'
+import originalContacts from './contacts.json'
 
 function App() {
+  const firstFive = originalContacts.slice(0, 50)
```

Comment:
Prob you type an exta 0
```suggestion
  const firstFive = originalContacts.slice(0, 5)
```